arches_component_lab/src/arches_component_lab/generics/GenericFormField.vue

```diff
@@ -63,6 +72,26 @@ function internalResolver(event: FormFieldResolverOptions) {
 
     return value;
 }
+
+function findFormFieldNode(node?: VNode): VNode | undefined {
```

Comment:
Hmmm.... explicitness vs 🪄 

Should we _not_ require this magic `form-input` attr in cases of only 1 child input?

Reply:
happy to chat about this any time, we def need to land on a pattern sooner rather than later